src/metabase/upload.clj

```diff
@@ -136,6 +164,15 @@
         ", " #"\d[\d \u00A0]*"
         ".’" #"\d[\d’]*"))))
 
+(defn- float-or-int-regex [number-separators]
+  (with-parens
+   (with-currency
+    (case number-separators
+      ("." ".,") #"\d[\d,]*\.0+"
+      ",." #"\d[\d.]*\,[0]+"
+      ", " #"\d[\d \u00A0]*\,[0.]+"
+      ".’" #"\d[\d’]*\.[0.]+"))))
```

Comment:
I don't know why we allow additional decimal points, this might be duplicating a bug.

Reply:
Was just about to ask that. L173 is also curious (can you really write Pi as `3,141.592[etc]`?)